lib-rca/src/main/java/com/facebook/openwifi/librca/models/InputStats.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifi.librca.models;
+
+/** Input Data Model */
+public class InputStats {
+    public static class RadioConfig {
+        public int channel;
+        public int channelWidth;
+        public int txPower;
+        public String phy;
+    }
+    public class Association{
+        /** Rate information for receive/transmit data rate*/
+        public class Rate {
+            public long bitRate;
+            public int chWidth;
+            public int mcs;
+        }
+        public long connected;
+        public long inactive;
+        public int rssi;
+        public long rxBytes;
+        public long rxPackets;
+        public Rate rxRate;
+        public long txBytes;
+        public long txDuration;
+        public long txFailed;
+        public long txPackets;
+        public Rate txRate;
+        public long txRetries;
+        public int ackSignal;
+        public int ackSignalAvg;
+        public long txPacketsCounters;
+        public long txErrorsCounters;
+        public long txDroppedCounters;
+        public long activeMsRadio;
+        public long busyMsRadio;
```

Comment:
Maybe add a comment here as well mentioning that the metrics below are from the radio the client was associated to